docs/components/ProgressIndicator.md

```diff
@@ -245,8 +245,7 @@ In the layout:
 ```xml
 <com.google.android.material.progressindicator.LinearProgressIndicator
     android:layout_width="match_parent"
```

Comment:
```suggestion
<com.google.android.material.progressindicator.CircularProgressIndicator
    android:layout_width="wrap_content"
```

Reply:
Sorry I didn't catch that earlier! Should be good now.